tests/plugins/test_compare_components.py

```diff
@@ -175,3 +197,46 @@ def test_bad_component_type(tmpdir):
 
     with pytest.raises(PluginFailedException):
         runner.run()
+
+
+@pytest.mark.parametrize('mismatch', (True, False))
+def test_mismatch_reporting(tmpdir, caplog, mismatch):
+    """Test if expected log entries are reported when compoentns mismatch"""
```

Comment:
Typo: components

Reply:
Fixed.

I've changed settings in my IDE to mark typos in way I couldn't ignore :-)